gateway/1.5/modules/ROOT/pages/policies-pdk-template-caching-policy.adoc

```diff
@@ -0,0 +1,424 @@
+= Data Caching Policy Template
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Use the data caching policy template as an example of how to xref:policies-pdk-configure-features.adoc#share-data-between-workers-and-caching.adoc[configure policy caching].
+
+== Before You Begin
+
+. xref:policies-pdk-prerequisites.adoc[]
+
+== Policy Use Case
+
+An antique store that buys and sells items needs to track the stock of each item in its collection and who buys it. To track the inventory movement, the company uses an API where each unique path represents a different item.. 
+
+During business hours, the catalog changes constantly and caching the available items isn't very useful. Outside of business hours, the stock is frozen, and the catalog can be cached without any concern. Given this, the manager of the antique store's server decides to implement a caching policy that caches the stock during the non-business hours. 
```

Comment:
```suggestion
During business hours, the catalog changes constantly and caching the available items isn't very useful. Outside of business hours, the stock is frozen, and the catalog can be cached without any concern. Given this, the antique store's server manager decides to implement a caching policy that caches the stock during the non-business hours. 
```

Reply:
I'm not sure about "isn't very useful" and "Given this" for global language and conciseness.
Could you simplify something like:
Because the store's catalog items in stock change constantly during business hours but are frozen outside of business hours, the server manager decides to implement a caching policy that caches the stock during non-business hours.